waveshare2in13v2/example_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package waveshare2in13v2_test
+
+import (
+	"image"
+	"log"
+
+	"periph.io/x/conn/v3/spi/spireg"
+	"periph.io/x/devices/v3/ssd1306/image1bit"
+	"periph.io/x/devices/v3/waveshare2in13v2"
+	"periph.io/x/host/v3"
+)
+
+func Example() {
+	// Make sure periph is initialized.
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// Use spireg SPI bus registry to find the first available SPI bus.
+	b, err := spireg.Open("")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer b.Close()
+
+	dev, err := waveshare2in13v2.NewHat(b, &waveshare2in13v2.EPD2in13v2) // Display config and size
+	if err != nil {
+		log.Fatalf("failed to initialize epd: %v", err)
+	}
+
+	// Draw on it. Black text on a white background.
+	img := image1bit.NewVerticalLSB(dev.Bounds())
+	// Note: this code is commented out so periph does not depend on:
```

Comment:
This is not a problem now that this package uses go modules. So we should revisit these examples and use golang.org/x/image when it makes sense. So feel free to uncomment if you want.

Reply:
Done